Gemfile

```diff
@@ -149,7 +149,7 @@ group :test do
   gem 'capybara', "~> 2.2.1"
   gem 'cucumber-rails', '~> 1.4.0', :require => false
   gem 'cucumber'
-  gem 'selenium-webdriver', '~> 2.45.0'
```

Comment:
What about `~> 2.45`? That would allow updating to 2.46, 2.47, 2.48 etc, but it would not allow to update to 3.0.